src/TLuaInterpreter.cpp

```diff
@@ -14594,6 +14594,26 @@ int TLuaInterpreter::getOS(lua_State* L)
     return 1;
 }
 
+int TLuaInterpreter::getClipboardText(lua_State* L)
```

Comment:
Please add link to documentation in wiki

Reply:
Done